pkg/commands/workload.go

```diff
@@ -771,7 +772,47 @@ func isUrl(str string) (bool, error) {
 	}
 }
 
-func (opts *WorkloadOptions) GetReadyConditionWorkers(c *cli.Config, workload *cartov1alpha1.Workload, workers []wait.Worker) []wait.Worker {
```

Comment:
Refer to [this comment,](https://github.com/vmware-tanzu/apps-cli-plugin/pull/533#discussion_r1176806070) all these helper funcs were changed to not be part of `WorkloadOptions` since this struct is not responsible for workers or race conditions.